libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -382,6 +385,23 @@ MAV_RESULT AP_Mount_Backend::handle_command_do_gimbal_manager_configure(const ma
             break;
     }
 
+    // send gimbal_manager_status if control has changed
+    if (prev_control_id_sysid != mavlink_control_id.sysid || prev_control_id_compid != mavlink_control_id.compid) {
+        uint32_t flags = GIMBAL_MANAGER_FLAGS_ROLL_LOCK | GIMBAL_MANAGER_FLAGS_PITCH_LOCK;
+        if (_yaw_lock) {
+            flags |= GIMBAL_MANAGER_FLAGS_YAW_LOCK;
+        }
+        const mavlink_gimbal_manager_status_t manager_status_packet{
```

Comment:
It would be nice if we could instead re-use the code that we have in AP_Mount_Backend::send_gimbal_manager_status().

If we're happy with the gimbal-manager-status going out to only the channel that we received the do-gimbal-manager-configure command on then we could pass the channel into handle_command_do_gimbal_manager_configure() and reuse it.  If instead we'd like all mavlink channels to get the message then we could call gcs().send_message(MSG_GIMBAL_MANAGER_STATUS);

Reply:
Thank you! I just saw the other PR, sorry I just saw this now.